java/src/jmri/jmrit/display/layoutEditor/ConnectivityUtil.java

```diff
@@ -1923,6 +1923,29 @@ public Block getExitBlockForTrackNode(
     }
 
     // support methods
+
+    /**
+     * Provide the "neither branch leads to next block" warning message if relevant
+     */
+    private void neitherBranchWarning(LayoutTurnout layoutTurnout, LayoutBlock nextLayoutBlock, boolean suppress) {
+        if (!suppress) {
+            String layoutBlockInfo = layoutTurnout.toString();
```

Comment:
`layoutTrackInfo` or `layoutTurnoutInfo` would be a better variable name.

Reply:
Done,  Thanks!